DontForget/Sources/Data/Network/EndPoint.swift

```diff
@@ -57,20 +57,25 @@ extension DontForgetTarget: TargetType {
     
     var task: Moya.Task {
         switch self {
-        case .registerAnniversary(let request):
-            return .requestJSONEncodable(request as! Encodable)
+        case .registerAnniversary(let deviceUuid, let title, let date, let content, let type, let alarmSchedule):
```

Comment:
```suggestion
        case let .registerAnniversary(deviceUuid, title, date, content, type, alarmSchedule):
```
이렇게 쓰면 조금 더 간단하게 쓸 수 있겠네요!

Reply:
1. 괄호 개행
-> 넵! 죄송함다!
2. 백그라운드 색 말씀하시는거죠? 확인 바로하겠슴다! 그 아마 처음에 포커스를 무조건 주고 키보드 높이로 색변경을 줘서그런가봐용!
3. 사랑하는 엄마에게 이거말씀하시는거죠? 이거 성규형이 그냥 디폴트값으로 넣어라 하더라구요. 저도 물어봤는데 이상하지만 PM님 의견이 확고하여 그리했슴당!